.github/workflows/pylint.yml

```
@@ -0,0 +1,26 @@
+name: Pylint
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7"]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v11.5
```

```suggestion
    - name: Get changed files
      id: changed-files
      uses: tj-actions/changed-files@v11.5
```

indentation is a bit off